lib/engine/game/g_1840/round/line_operating.rb

```diff
@@ -18,6 +18,15 @@ def self.short_name
           def name
             'Line Round'
           end
+
+          def after_process(action)
+            entity = @entities[@entity_index]
+            case action
```

Comment:
not sure you need a case here


process_action if action.is_a?(x) && !action.routes.empty?